app/src/main/java/cto/its/ny/gov/fakenews/views/adapter/ArticleViewListAdapter.kt

```diff
@@ -0,0 +1,35 @@
+package cto.its.ny.gov.fakenews.views.adapter
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import cto.its.ny.gov.fakenews.R
+import cto.its.ny.gov.fakenews.model.Article
+import cto.its.ny.gov.fakenews.util.services.NewsService
+import cto.its.ny.gov.fakenews.views.holder.ArticleViewListHolder
+
+class ArticleViewListAdapter(val clickListener: ArticleClickListener): RecyclerView.Adapter<ArticleViewListHolder>() {
+    private var arrayOfArticles = NewsService().createArticles(0 .. 9)
```

Comment:
While this works now, you'll want to eventually provide the list of articles in to the constructor of `ArticleViewListAdapter`.  You won't want to encapsulate your `NewsService()` instance inside here as it restricts fetching the articles to the instantiation of this adapter.